app/views/admin/world_locations/show.html.erb

```diff
@@ -7,7 +7,7 @@
       <span class="name"><%= @world_location.name %></span>
       &ldquo;<span class="title"><%= @world_location.title %></span>&rdquo;
     </h1>
-    <p><%= view_on_website_link_for @world_location %></p>
+    <p><%= link_to "View on website", "/world/#{@world_location.name}/news" %></p>
```

Comment:
I think this needs to be `@world_location.slug` in order return the correct URL. Calling `.name` for example would return "São Tomé and Principe" whereas `.slug` would return "sao-tome-and-principe"